options.c

```diff
@@ -4782,6 +4782,11 @@ static char *bc_calc(char *str)
 	memcpy(buf, str, tmp - str);
 	free(str);
 	return strdup(buf);
+#else /* __rtems__ */
+	log_err("fio: performing math is not supported\n");
+	printf("fio: performing math is not supported\n");
+	return ' ';
```

Comment:
That might isn't a good idea. It will return a pointer to a string where it isn't entirely clear were it comes from. That could theoretically be on the task stack. Either use a pointer to a `static const char[] = ' ';` or even better: return NULL like in other error cases. If there is math in the string, we can't do what the user want anyway. So an error is OK.